test/meuse_comp.jl

```diff
@@ -0,0 +1,54 @@
+@testset "Meuse R gstat compatibility" begin
+    """meuse_grid.csv constains (x, y) plus estimation/variance of log(zinc)
+    for global and local (25 samples)
+    """
+    datadir = joinpath(@__DIR__,"data")
+    data_meuse = readgeotable(joinpath(datadir,"meuse.csv"), coordnames=(:x,:y))
+    grid_meuse = readgeotable(joinpath(datadir,"meuse_grid.csv"), coordnames=(:x,:y))
+
+    dim = 2
+    tol = 0.001
+    knn = 25
+
+    vmodel = SphericalVariogram(range=874.0,nugget=0.04,sill=0.59+0.04)
+    problem = EstimationProblem(data_meuse, grid_meuse, :log_zinc)
+    
+    @testset "LocalKriging" begin
+
+        local_kriging = Kriging(
+            :log_zinc => (variogram=vmodel,maxneighbors=knn)
+        )
+
+        solution = solve(problem, local_kriging)
+        est_local, var_local = solution[:log_zinc]
+
+        pdomain = domain(problem)
+        xₒ = Array{coordtype(pdomain),}(undef,dim)
+        for i in 1:nelms(grid_meuse)
+            coordinates!(xₒ, pdomain, i)
```

Comment:
xo doesnt seem to be used, can you please erase it?

Reply:
Sure. It was used in early stages. I'm missing some warning from the compiler, such as unused variables :-)

Reply:
Thanks, it is good to review the code carefully before the PR is ready for review. You can also signal the PR as draft on GitHub so that we only start the review process after it is ready for review.

Reply:
It is good indeed.